docs/example-formik.md

```diff
@@ -14,9 +14,9 @@ import { Formik, Field, Form } from 'formik'
 const sleep = ms => new Promise(r => setTimeout(r, ms))
 
 export const MyForm = ({ onSubmit }) => {
-  const handleSubmit = async values => {
+  const handleSubmit = async (values, formikBag) => {
```

Comment:
For future reference, could you point to forkik documentation for this? I couldn't find it.

Reply:
Do you mean the Formik doc for the onSubmit callback? If so then this is the doc - https://formik.org/docs/api/formik#onsubmit-values-values-formikbag-formikbag--void--promiseany

Thanks.

Reply:
We just need to revert https://github.com/testing-library/testing-library-docs/pull/692.

You probably don't want to expose the formik bag anyway.